libtsuba/src/tsuba.cpp

```diff
@@ -205,6 +207,108 @@ tsuba::ListAvailableViews(
   return std::make_pair(latest_version, views_found);
 }
 
+katana::Result<std::vector<std::pair<katana::Uri, katana::Uri>>>
+tsuba::CreateSrcDestFromViewsForCopy(
+    const std::string& src_dir, const std::string& dst_dir, uint64_t version) {
+  std::vector<std::pair<katana::Uri, katana::Uri>> src_dst_files;
+
+  // List out all the files in a given view
+  auto rdg_views = KATANA_CHECKED(tsuba::ListAvailableViews(src_dir, version));
+  for (const auto& rdg_view : rdg_views.second) {
+    KATANA_LOG_WARN("view_path: {}", rdg_view.view_path);
+    auto uri = KATANA_CHECKED(katana::Uri::Make(src_dir));
+
+    auto rdg_manifest_res =
+        tsuba::RDGManifest::Make(uri, rdg_view.view_type, version);
+    if (!rdg_manifest_res) {
+      continue;
+    }
+
+    auto fnames = KATANA_CHECKED(rdg_manifest_res.value().FileNames());
+    for (auto fname : fnames) {
+      auto src_file_path = katana::Uri::JoinPath(src_dir, fname);
+      auto src_file_uri = KATANA_CHECKED(katana::Uri::Make(src_file_path));
+
+      // If we are a manifest, we need to change our version to 1.
```

Comment:
I don't understand this comment. It suggests that some RDGs don't have manifests and it that case we would keep the existing version, but we always set the version to 1, so this comment seems out of place.